platform/mds/mds/src/main/java/org/motechproject/mds/repository/MotechDataRepository.java

```diff
@@ -70,6 +70,12 @@ public PersistenceManager getPersistenceManager() {
         return pm;
     }
 
+    public void evictAll() {
+        if (persistenceManagerFactory != null) {
+            persistenceManagerFactory.getDataStoreCache().evictAll();
```

Comment:
should we evict everything? Can we add add a second method that evicts the records for this entity only, not the entire cache?